### PR TITLE
Update config.ini with identity migration options

### PR DESCRIPTION
**Pull request by roman-verchikov**
Adds description for `optimize_user_role_fetch` config option.

### CODE REVIEW

File: configs/config.ini [hunk 1]
```diff
@@ -158,6 +158,11 @@ path_scenario = scenario/migrate.yaml
 # rollback.
 mysqldump_host = <specific MySQL DB hostname or IP address, not VIP>
 
+# Uses low-level DB requests if set to True, may be incompatible with more
+# recent versions of Keystone. Tested on grizzly, icehouse and juno. True by
+# default
+optimize_user_role_fetch = [True|False]
+
```
Comment (miarmak) [inline configs/config.ini]: We still have one more config template in `devlab`... Maybe we need to add this option there too?

**Comment by miarmak** [comment]
Looks good overall. Only 1 question inline.

File: configs/config.ini (hunk 1 above)
Comment (Settis) [inline configs/config.ini]: Agree. It can speedup our CI.

**Comment by Settis** [comment]
+1

File: configs/config.ini (hunk 1 above)
Comment (roman-verchikov) [inline configs/config.ini]: `optimize_user_role_fetch` is enabled by default, so it's already there.

**Comment by roman-verchikov** [comment]
can be merged

**Pull request merged**